mlflow/java/client/pom.xml

```diff
@@ -74,6 +82,7 @@
               <include>commons-codec:commons-codec</include>
               <include>commons-logging:commons-logging</include>
               <include>org.apache.httpcomponents:httpcore</include>
+              <include>org.yaml:snakeyaml</include>
```

Comment:
Do we actually use YAML? Is it a dependency of commons-io or ini4j?

Reply:
Sorry, good catch. Prior implementation.